src/mlpack/methods/random_forest/random_forest_impl.hpp

```diff
@@ -344,13 +366,15 @@ void RandomForest<
 
 template<
     typename FitnessFunction,
+    bool UseBootstrap,
```

Comment:
Personally, I think a user will use this parameter less often than other template parameters, so I might suggest putting it last.  (Also that means there is no reverse incompatibility. :))

Reply:
I don't know how, but I always mess up with this sequence of templates XD.

I will fix this 👍